lib/config/decrypt.ts

```diff
@@ -134,6 +134,12 @@ export async function decryptConfig(
   for (const [key, val] of Object.entries(config)) {
     if (key === 'encrypted' && is.object(val)) {
       logger.debug({ config: val }, 'Found encrypted config');
+
+      const encryptedWarning = GlobalConfig.get('encryptedWarning');
+      if (is.string(encryptedWarning)) {
+        logger.once.warn(`Found encrypted config, ${encryptedWarning}`);
```

Comment:
```suggestion
        logger.once.warn(encryptedWarning);
```

Let's log the encryptedWarning text exactly as given

Reply:
i have no indication that this message is printed when i create the PR, so i needed a static string to match in my `RepoProblems` in order to show warnings in PR.
look for `if (entry.startsWith('WARN: Found encrypted config')) {`
or should i look for the keyword `WARN:` and write that Warnings were raised regardless of this specific flag?

Reply:
then Warnings will appear in PR as they appear in `repoProblems`,
ill have this in a separate PR if thats what u want

Reply:
if u want pr warnings to appear for any warning that goes to repoProblems there you go
https://github.com/renovatebot/renovate/pull/29205

note: this might change current behaviour for some users.